packages/client/src/Conversation.tsx

```diff
@@ -79,14 +84,7 @@ export default function Conversation({ accountId, conversationId }: Props) {
     accountId: accountId!,
     conversationId: conversationId!
   })
-  const [setIsRead, setIsReadResult] = graphql.useSetIsReadMutation({
-    variables: { conversationId: conversationId!, isRead: true }
-  })
-  React.useEffect(() => {
-    if (data && data.conversation && !data.conversation.isRead) {
-      setIsRead()
-    }
-  }, [data, setIsRead])
+  const setIsReadResult = useSetIsRead(data && data.conversation)
```

Comment:
I decided to simplify the code here by replacing the graphql mutation and effect hook with a single invocation of a custom hook.